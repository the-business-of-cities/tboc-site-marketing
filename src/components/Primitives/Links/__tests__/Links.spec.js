import { MaybeLink } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("MaybeLink", () => {
  it("renders default correctly", () => {
    expect(shallow(<MaybeLink />)).toMatchSnapshot();
  });

  it("renders gatsby link correctly", () => {
    expect(shallow(<MaybeLink to="some-url"/>)).toMatchSnapshot();
  });

  it("renders href correctly", () => {
    expect(shallow(<MaybeLink href="some-other-url"/>)).toMatchSnapshot();
  });
});
