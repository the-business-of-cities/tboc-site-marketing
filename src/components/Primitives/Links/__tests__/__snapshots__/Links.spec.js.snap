// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaybeLink renders default correctly 1`] = `<a />`;

exports[`MaybeLink renders gatsby link correctly 1`] = `
<ForwardRef
  to="some-url"
/>
`;

exports[`MaybeLink renders href correctly 1`] = `
<a
  href="some-other-url"
/>
`;
